### ISSUE TITLE
UnityException: JDK not found on 2019.4.20f1-android-0

### ISSUE THREAD

**Issue by jim-kirisame** [game-ci/docker#94]
**Bug description**
Cannot build on Image: 2019.4.20f1-android-0 (15c873cd536d):

```
.......
Disconnect from CacheServer
Refreshing native plugins compatible for Editor in 1.73 ms, found 0 plugins.
Preloading 0 native plugins for Editor in 0.01 ms.
DisplayProgressbar: Checking prerequisites
Android PostProcess task "Checking prerequisites" took 21.5857 ms
DisplayProgressbar: Detect Java Development Kit (JDK)
UnityException: JDK not found
Java Development Kit (JDK) directory is not set or invalid. Please, fix it in Preferences -> External Tools

UnityEditor.BuildPipeline:BuildPlayerInternalNoCheck(String[], String, String, BuildTargetGroup, BuildTarget, BuildOptions, Boolean)
UnityEditor.BuildPipeline:BuildPlayerInternal(String[], String, String, BuildTargetGroup, BuildTarget, BuildOptions) (at /home/bokken/buildslave/unity/build/Editor/Mono/BuildPipeline.bindings.cs:385)
UnityEditor.BuildPipeline:BuildPlayer(String[], String, String, BuildTargetGroup, BuildTarget, BuildOptions) (at /home/bokken/buildslave/unity/build/Editor/Mono/BuildPipeline.bindings.cs:284)
UnityEditor.BuildPipeline:BuildPlayer(BuildPlayerOptions) (at /home/bokken/buildslave/unity/build/Editor/Mono/BuildPipeline.bindings.cs:258)
UnityEditor.BuildPipeline:BuildPlayer(String[], String, BuildTarget, BuildOptions) (at /home/bokken/buildslave/unity/build/Editor/Mono/BuildPipeline.bindings.cs:253)
CustomBuilder.Editor.Builder:GenericBuild(String[], String, BuildTarget, BuildOptions) (at Assets/Plugins/CustomBuilder/Editor/Builder.cs:98)
CustomBuilder.Editor.Builder:GenericBuild(BuildConfig) (at Assets/Plugins/CustomBuilder/Editor/Builder.cs:119)
CustomBuilder.Editor.Builder:BuildFromArg() (at Assets/Plugins/CustomBuilder/Editor/Builder.cs:166)
 
(Filename: Assets/Plugins/CustomBuilder/Editor/Builder.cs Line: 98)

DisplayProgressbar: 
Unloading 89 Unused Serialized files (Serialized files now loaded: 0)
System memory in use before: 58.1 MB.
System memory in use after: 58.2 MB.

Unloading 54 unused Assets to reduce memory usage. Loaded Objects now: 3552.
Total: 24.139994 ms (FindLiveObjects: 0.821163 ms CreateObjectMapping: 0.442969 ms MarkObjects: 20.875828 ms  DeleteObjects: 1.940240 ms)

Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
Loaded scene 'Temp/__Backupscenes/0.backup'
	Deserialize:            5.378 ms
	Integration:            95.039 ms
	Integration of assets:  0.051 ms
	Thread Wait Time:       15.008 ms
	Total Operation Time:   115.476 ms
System memory in use before: 56.6 MB.
System memory in use after: 56.8 MB.

Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 3551.
Total: 18.220428 ms (FindLiveObjects: 0.791386 ms CreateObjectMapping: 0.556141 ms MarkObjects: 16.734643 ms  DeleteObjects: 0.106361 ms)

Exception: BuildPlayer failure: 2 Errors.
  at CustomBuilder.Editor.Builder.GenericBuild (System.String[] scenes, System.String target_dir, UnityEditor.BuildTarget build_target, UnityEditor.BuildOptions build_options) [0x0010e] in /root/project/Assets/Plugins/CustomBuilder/Editor/Builder.cs:110 
  at CustomBuilder.Editor.Builder.GenericBuild (CustomBuilder.Editor.BuildConfig config) [0x00001] in /root/project/Assets/Plugins/CustomBuilder/Editor/Builder.cs:119 
  at CustomBuilder.Editor.Builder.BuildFromArg () [0x0002b] in /root/project/Assets/Plugins/CustomBuilder/Editor/Builder.cs:166 
 
(Filename: Assets/Plugins/CustomBuilder/Editor/Builder.cs Line: 110)

executeMethod method CustomBuilder.Editor.Builder.BuildFromArg threw exception. 
```

**How to reproduce**

Just build an Android project.

**Expected behavior**

Successfully build.

**Additional details**

Is this bug related to #76 ?

**Comment by jim-kirisame** [comment]
I have try 2019.4.22f1 but it still not works.

**Comment by webbertakken** [comment]
This was solved in https://github.com/game-ci/docker/pull/90 and will be released in `0.11` of https://github.com/game-ci/docker/releases

I'm closing this, but once 0.11 is out, if it still doesn't work, don't hesitate to comment here or open a new issue.

**Comment by jim-kirisame** [comment]
It doesn't work, and the error is the same.

The image I use is 2019.4.22f1-android-0, Image ID: 48126b4bac7f, digest: 6cffda75593
